src/d3a/d3a_core/redis_connections/redis_communication.py

```diff
@@ -126,6 +133,21 @@ def _slowdown_callback(self, message):
             return
         self._simulation.slowdown = slowdown
 
+    def _live_event_callback(self, message):
+        data = json.loads(message["data"])
+        try:
+            self._live_events.add_event(data)
```

Comment:
AFAIS, there is no `self._live_events` yet. This is the placeholder for the deserialisation of the events, right?

Reply:
This PR is based on branch `feature/D3ASIM-2332` , where the live events are actually implemented in d3a and such an object exists.